rwengine/src/render/GameRenderer.cpp

```diff
@@ -357,7 +357,11 @@ void GameRenderer::renderWorld(GameWorld* world, const ViewCamera& camera,
     std::sort(renderList.begin(), renderList.end(),
               [](const Renderer::RenderInstruction& a,
                  const Renderer::RenderInstruction& b) {
-                  return a.sortKey < b.sortKey;
+                    if (!a.drawInfo.blend && b.drawInfo.blend) return true;
+                    if (!b.drawInfo.blend && a.drawInfo.blend) return false;
+                    if (a.sortKey > b.sortKey) return true;
```

Comment:
Won't this sort opaque objects into the wrong order?

Reply:
ah, yes ;)